wagn/lib/wagn/parser.rb

```diff
@@ -0,0 +1,93 @@
+# -*- encoding : utf-8 -*-
+
+module Wagn
+  class Parser
+    class << self
+      def wagn envs
```

Comment:
Unused method argument - `envs`. If it's necessary, use `_` or `_envs` as an argument name to indicate that it won't be used. You can also write as `wagn(*)` if you want the method to accept any arguments but don't care about them.<br>Method has too many lines. [21/15]